src/morello-install/README.md

```diff
@@ -671,29 +712,27 @@ installation process:
 
 ```
 ...
-Mounting late filesystems:.
-Security policy loaded: MAC/ntpd (mac_ntpd)
-Starting ntpd.
-Starting cron.
-Starting sendmail_submit.
-Starting sendmail_msp_queue.
 Performing sanity check on sshd configuration.
 Starting sshd.
+Starting cron.
 Starting background file system checks in 60 seconds.
+Starting sddm.
 
-Sun May  8 00:13:04 BST
-CheriBSD/arm64 (cheribsd) (ttyu0)
+Fri Dec  8 11:32:45 UTC
+CheriBSD/arm64 (cheri-blossom.sec.cl.cam.ac.uk) (ttyu0)
 
 login: root
 Password:
-May  8 00:13:25 cheribsd login[772]: ROOT LOGIN (root) ON ttyu0
-Last login: Sat May  7 23:41:04 on ttyu0
-FreeBSD 14.0-CURRENT #0 dev-d4897febcde: Tue May  3 13:34:49 BST 2022     rnw24@zeno.sec.cl.cam.ac.uk:/home/rnw24/cheri/build/cheribsd-morello-purecap-build/home/rnw24/cheri/cheribsd/arm64.aarch64c/sys/GENERIC-MORELLO
+Dec  8 11:37:01 cheri-blossom login[1277]: ROOT LOGIN (root) ON ttyu0
+FreeBSD 14.0-CURRENT aarch64c 1400094 (GENERIC-MORELLO) #0 dev-131fb8e41f70: Fri Dec  8 03:23:10 UTC 2023
```

Comment:
This should be updated to use the actual 23.11 uname string, since the branch, hash and build time are all different

Reply:
Hmm. I’m not set up to easily do a fresh install from here. Anyone else have the string handy and could do this update? (@gvnn3? @kwitaszczyk?)